sources/api/pluto/src/main.rs

```diff
@@ -5,22 +5,47 @@
 
 pluto is called by sundog to generate settings required by Kubernetes.
 This is done dynamically because we require access to dynamic networking
-setup information.
+and cluster setup information.
 
-It makes calls to IMDS to get meta data:
+It uses IMDS to get information such as:
 
-- Cluster DNS
+- Instance Type
 - Node IP
+
+It uses EKS to get information such as:
+
+- Service IPV4 CIDR
+
+It uses the Bottlerocket API to get information such as:
+
+- Kubernetes Cluster Name
+- AWS Region
+
+# Interface
+
+Pluto takes the name of the setting that it is to generate as its first
+argument.
+It returns the generated setting to stdout as a JSON document.
+Any other output is returned to stderr.
+
+Pluto returns a special exit code of 2 to inform `sundog` that a setting should be skipped. For
+example, if `max-pods` cannot be generated, we want `sundog` to skip it without failing since a
+reasonable default is available.
 */
 
-use reqwest::blocking::Client;
+mod api;
+mod eks;
+
+use api::{get_cluster_name, get_region};
+use eks::get_cluster_cidr;
```

Comment:
```suggestion
```
nit: I think it'd be clearer to use the whole `eks::get_cluster_cidr()` below, to make it clearer that it's not a local function, since there a few similarly named functions here.